src/schema-gen/InspectorXSLT/testing/current/tiny_metaschema_COMPOSED-metaschema.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Non-blocking: is there a schema or formal spec for the composed Metaschema variant? From experience, I know what it is, but I do not know how to validate it or review new content in the intermediate composed form, aside from assuming modified tools in PR under review are correct or "good enough."

Reply:
It is an awesome question, and one that has been on my come-back-to list for an awfully long time.

Short answer - no, but not because it's not a good idea - if anything, it's because neither the composed Metaschema nor the 'supermodel' format related to it have been considered stable enough - i.e. this work is related to shaking out both of those. An Issue on this could be warranted as a long-term placeholder if modeling the internals (a metaschema for a composed metaschema) is of interest to you.

Reply:
That's ok, but it is never really clear if the file is kept as an intermediate transpiled output why you would want to keep and cause more noise amongst signal, especially if you want to rebuild it every time. I would not include it add it to `.gitignore` like I do not want the transpiled Javascript `.js` files in a Typescript project, `.pyc` files of Python bytecode for Python, et cetera.